docs/nightly-alpha/advanced_syntax/builtins.md

```
@@ -58,23 +80,50 @@ echo(len(lines))
 
 > WARNING: While embedded Bash commands like `$ cat foo.txt $` require a `trust` keyword or `failed` block, Amber does not currently support this for the `lines` builtin. If the file does not exist at runtime, the program will terminate midway, **potentially losing data stored only in variables**
 
+## Cp
+
+Copy files or directories from location to another. This builtin is failable.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)